Tools/clinic/clinic.py

```diff
@@ -2862,6 +2862,9 @@ def __getattr__(self, attr):
                     f"Note: accessing self.function inside converter_init is disallowed!"
                 )
             return super().__getattr__(attr)
+    # this branch is just here for coverage reporting
+    else:  # pragma: no cover
+        pass
```

Comment:
coverage was reporting only partial branch coverage for the `if not TYPE_CHECKING` branch immediately above without this, since `TYPE_CHECKING` is always `False` at runtime